### PR TITLE
Rename "key" -> "password" for passphrase encryption - Closes #1923

### LINKED ISSUE #1923: Rename "key" -> "password" for passphrase encryption
### Expected behavior

The secret ingredient you need to encrypt/decrypt a passphrase in the current implementation is a "password", not a "key". All namings should reflect that.

### Actual behavior

At some places, it is still called "key" which is at least misleading and at worst the point where users mess up with security.

Remaining "key"s are at least

- [x] the field name `key` in test/data/genesis_delegates.json
- [x] the field name `key` in test/unit/logic/delegate.js
- [x] the field name `decryptionKey` in https://app.swaggerhub.com/apis/LiskHQ/Lisk/1.0.26#/Node/updateForgingStatus

### Which version(s) does this affect? (Environment, OS, etc...)

Current development / Swagger 1.0.26

### PR DESCRIPTION
**Pull request by 4miners**
### What was the problem?
The secret ingredient you need to encrypt/decrypt a passphrase in the current implementation is a "password", not a "key". All namings should reflect that.
### How did I fix it?
Rename:
- `password` -> `passphrase` (fixtures)
- `key` -> `password`
- `defaultKey` -> `defaultPassword`
- `decryptionKey` -> `password`
### How to test it?
Run test suite.
### Review checklist

* The PR solves #1923
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: test/common/utils/random.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ random.account = function() {
 random.transaction = function(offset) {
 	return lisk.transaction.transfer({
 		amount: 1,
-		passphrase: accountFixtures.genesis.password,
+		passphrase: accountFixtures.genesis.secret,
```
Comment (willclarktech) [inline test/common/utils/random.js]: As above: why not `passphrase`?

File: test/fixtures/accounts.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const accounts = {};
 accounts.existingDelegate = {
 	address: '10881167371402274308L',
 	publicKey: 'addb0e15a44b0fdc6ff291be28d8c98f5551d0cd9218d749e30ddb87c6e31ca9',
-	password:
+	secret:
```
Comment (willclarktech) [inline test/fixtures/accounts.js]: As above.

File: test/functional/http/put/node.js [hunk 3]
```diff
@@ -79,7 +79,7 @@ describe('PUT /node/status/forging', () => {
 	it('using invalid key should fail', () => {
```
Comment (willclarktech) [inline test/functional/http/put/node.js]: Description should be updated.

File: test/functional/http/put/node.js [hunk 4]
```diff
@@ -79,7 +79,7 @@ describe('PUT /node/status/forging', () => {
...
 		var params = {
 			publicKey: validDelegate.publicKey,
-			decryptionKey: 'invalid key',
+			password: 'invalid key',
```
Comment (willclarktech) [inline test/functional/http/put/node.js]: `invalid password`

File: test/unit/modules/node.js [hunk 5]
```diff
@@ -131,7 +131,7 @@ describe('node', () => {
 			it('should return error with non delegate account', done => {
 				node_module.internal.toggleForgingStatus(
 					accountFixtures.genesis.publicKey,
-					accountFixtures.genesis.password,
+					accountFixtures.genesis.secret,
```
Comment (willclarktech) [inline test/unit/modules/node.js]: This should actually be `password`, right?

File: test/common/utils/random.js [hunk 6]
```diff
@@ -117,7 +117,7 @@ random.account = function() {
 random.transaction = function(offset) {
 	return lisk.transaction.transfer({
 		amount: 1,
-		passphrase: accountFixtures.genesis.password,
```
Comment (nazarhussain) [inline test/common/utils/random.js]: @4miners I am confused here, issue describe to rename 

```
key -> password
```
It does not refer any change required for `passphrase`

Comment (willclarktech) [reply test/common/utils/random.js]: Now that `key` is named `password`, it's crucial to distinguish passphrases from passwords!

**Comment by 4miners** [comment]
@willclarktech Resolved, please check again.

**Pull request merged**